From 566d4fd4a07a92fa0023d29d2ad5477bad537288 Mon Sep 17 00:00:00 2001
From: Ben Hutchings <ben@decadent.org.uk>
Date: Sat, 19 Dec 2009 14:51:09 +0000
Subject: [PATCH] ia64: Include <linux/personality.h> header in <asm/fcntl.h>

The definition of force_o_largefile() references personality, but this
header does not include <linux/personality.h>.  This happens to work
for sys_open(), but may not for any other users.  In particular, aufs2
now uses this macro and so fails to build on ia64.
---
 arch/ia64/include/asm/fcntl.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/ia64/include/asm/fcntl.h b/arch/ia64/include/asm/fcntl.h
index 1dd275d..3cac8f9 100644
--- a/arch/ia64/include/asm/fcntl.h
+++ b/arch/ia64/include/asm/fcntl.h
@@ -5,6 +5,8 @@
  *	David Mosberger-Tang <davidm@hpl.hp.com>, Hewlett-Packard Co.
  */
 
+#include <linux/personality.h>
+
 #define force_o_largefile()	\
 		(personality(current->personality) != PER_LINUX32)
 
-- 
1.6.5.7

